components/script/dom/node.rs

```diff
@@ -2426,6 +2417,21 @@ impl Node {
                 .map_or(ns!(), |elem| elem.locate_namespace(prefix)),
         }
     }
+
+    /// If the given untrusted node address represents a valid DOM node in the given runtime,
+    /// returns it.
+    #[allow(unsafe_code)]
```

Comment:
Since you are touching this do you mind adding the "# Safety" section to the rustdoc? This will fix a clippy warning:

https://rust-lang.github.io/api-guidelines/documentation.html#function-docs-include-error-panic-and-safety-considerations-c-failure